tests/bson/bug0974-001.phpt

```diff
@@ -31,7 +31,7 @@ object(stdClass)#%d (%d) {
   }
 }
 object(stdClass)#%d (%d) {
-  [%r(0|"0")%r]=>
+  ["0"]=>
```

Comment:
There are some more occurrences below: `%r(1|"1")%r`. Sorry for not specifying in my original comment.

Reply:
Looked for any non-zero keys and replaced those as well.